version/version.go

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2019-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package version
+
+import (
+	"fmt"
+	"runtime/debug"
+	"time"
+)
+
+// VersionInfo contains version information about the binary
+type VersionInfo struct {
+	Commit    string    `json:"commit"`
+	BuildTime time.Time `json:"build_time"`
+	Modified  bool      `json:"modified"`
```

Comment:
non-blocking: I'd include the Go runtime version since it can be somewhat important at times.

Reply:
https://github.com/mattermost/mattermost-load-test-ng/pull/898/commits/29bab705aac1af3d1e86a90befec6beb562b5f90